### DISCUSSION TITLE
0.0.16

### DISCUSSION THREAD

**Discussion by Hellowlol** [custom-components/nordpool#450] [Announcements]
## What's Changed
* Fix version number in HA by @oleg-d in https://github.com/custom-components/nordpool/pull/424
* Fix Config import by @tjorim in https://github.com/custom-components/nordpool/pull/433
* Issue #438: Pin version of nordpool package by @arghle in https://github.com/custom-components/nordpool/pull/439
* Update __init__.py by @Hellowlol in https://github.com/custom-components/nordpool/pull/449
* Update README.md by @EvertJob in https://github.com/custom-components/nordpool/pull/351

## New Contributors
* @oleg-d made their first contribution in https://github.com/custom-components/nordpool/pull/424
* @tjorim made their first contribution in https://github.com/custom-components/nordpool/pull/433
* @arghle made their first contribution in https://github.com/custom-components/nordpool/pull/439
* @EvertJob made their first contribution in https://github.com/custom-components/nordpool/pull/351

**Full Changelog**: https://github.com/custom-components/nordpool/compare/0.0.15...0.0.16

<hr /><em>This discussion was created from the release <a href='https://github.com/custom-components/nordpool/releases/tag/0.0.16'>0.0.16</a>.</em>

**Comment by Nornode** [comment]
Super, Thanks for swift action and dedication to get this running again. 🙏
